pkg/apis/config/v1alpha3/types.go

```diff
@@ -126,6 +126,8 @@ type Networking struct {
 	// If DisableDefaultCNI is true, kind will not install the default CNI setup.
 	// Instead the user should install their own CNI after creating the cluster.
 	DisableDefaultCNI bool `yaml:"disableDefaultCNI,omitempty"`
+	// KubeProxyMode defines if kube-proxy should operate in iptables or ipvs mode
+	KubeProxyMode string `yaml:"kubeProxyMode,omitempty"`
```

Comment:
shouldn't API changes be added only to the latest version - v1alpha4?

Reply:
yes, we've held onto v1alpha3 for a while, it's been dropped now.

Reply:
Yes, thank you. Corrected here and removed from v1alpha3.

Also, in https://github.com/kubernetes-sigs/kind/pull/1595/files#diff-50194a7023ae768e6fec0e8c8b122fbbR293 I'm putting the config in ``ConfigTemplateAlphaV3`` and also the Beta ones. Should I remove from here and keep only for the Beta ones, assuming those are the Go templates that will be used, or it depends on which version of k8s I'm creating with KinD?

In this same subject, I didn't changed the ``ConfigTemplateAlphaV2`` so just need a direction here :)